pennylane/devices/__init__.py

```diff
@@ -84,6 +84,48 @@
     transforms.split_non_commuting
     transforms.hamiltonian_expand
 
+
+Modifiers
+---------
+
+The ``modifiers`` allow for the easy addition of default behavior to a device.
+
+.. currentmodule:: pennylane.devices.modifiers
+.. autosummary::
+    :toctree: api
+
+    convert_single_circuit_to_batch
+    simulator_tracking
+
+For example with a custom device we can add simulator-style tracking and the ability
+to handle a single circuit. See the documentation for each modifier for more details.
+
+.. code-block:: python
+
+    @simulator_tracking
+    @convert_single_circuit_to_batch
```

Comment:
Could we discuss having a shorter name? E.g., `@batch_support`

Reply:
I like `batch_support`

Reply:
I can see some potential confusion there.  Since this is for a device that has batch support but wants single circuit support.  We aren't "adding" batch support.

Reply:
maybe `auto_batch`, or something like that?

Reply:
`single_tape_support`?